### ISSUE TITLE
DevicePixelRatio fix broken for Firefox

### ISSUE THREAD

**Issue by wswebcreation** [koola/pix-diff#26]
**pix-diff version:** v1.0.12

After v1.0.12 the DevicePixelRatio fix for Firefox is broken and returns `undefined`. This has to do with this piece of code:

```
    // init
    browser.driver.manage().window().setSize(this.width, this.height)
        .then(function () {
            return browser.getProcessedConfig();
        })
        .then(function (data) {
            this.capabilities = data.capabilities;
            assert.ok(this.capabilities.browserName, 'Browser name is undefined.');
            this.browserName = this.capabilities.browserName.toLowerCase();
            // Require PixDiff matchers for jasmine(2)/mocha
            if (data.framework !== 'custom') {
                require(path.resolve(__dirname, 'framework', data.framework));
            }
            if (!this.isFirefox()) {
                return this.getPixelDeviceRatio();
            }
        }.bind(this))
        .then(function (ratio) {
            this.devicePixelRatio = ratio;
        }.bind(this));
```

I'll create a PR for this.

**Comment by koola** [comment]
@wswebcreation Fixed in commit baac6ad4aef166a4b577e3cb6193d37cd25fb45a

**Comment by wswebcreation** [comment]
Tnx
